cmake/msvc_version.c

```diff
@@ -0,0 +1 @@
+MSVC _MSC_FULL_VER
```

Comment:
I deleted this file in #290. Apparently, ctest still uses that to retrieve the exact MSVC version. However, this can also be done by just parsing the output of `cl.exe`. So here is a fix: #298. Please remove the commit that readds `msvc_version.c`. I really want that file to be gone :)

Reply:
I'll bring your commit in!